### ISSUE TITLE
Logical vectors don't recycle

### ISSUE THREAD

**Issue by rgdicker** [gdemin/expss#47]
The documentation for `count_if` says of the `...` arguments 

> Data on which criterion will be applied. Vector, matrix, data.frame, list. Shorter arguments will be recycled. 

However, it appears that if the shorter arguments are logical, they are not recycled

```
count_if(TRUE, c(TRUE, FALSE), TRUE)
# [1] 2
count_if(1, c(1, 0), 1)
# [1] 3
```

**Comment by gdemin** [comment]
Thanks, I will fix it in the next release.

**Comment by gdemin** [comment]
Fixed in version 0.10.1
